### PR TITLE
push to stable

### PR DESCRIPTION
**Pull request by vanhauser-thc**
IMHO we should make a release now before the documentation rewrite project is making changes, as until it ends I feel there could be too many inconsistencies.

So we should do the release  early next week.

WDYT?

### CODE REVIEW

**Comment by vanhauser-thc** [comment]
Is someone able to resolve the conflict?
I dont understand why we have a conflict every time we push.
And I am clueless how to resolve them if they cant be resolved via the web UI :-( (I don't undestand the hints github gives on how to do this via the cmdline ...)

File: docs/Changelog.md [hunk 1]
```diff
@@ -20,23 +20,28 @@ sending a mail to <afl-users+subscribe@googlegroups.com>.
...
+    - do honor AFL_LLVM_{ALLOW/DENY}LIST for LTO autodictionary and DICT2FILE
     - We do support llvm versions from 3.8 to 5.0 again
   - frida_mode:
     - several fixes for cmplog
     - remove need for AFL_FRIDA_PERSISTENT_RETADDR_OFFSET
+    - less coverage collision
     - feature parity of aarch64 with intel now (persistent, cmplog,
       in-memory testcases, asan)
-  - qemu_mode:
-    - performance fix when cmplog was used
```
Comment (domenukk) [inline docs/Changelog.md:33]: Has this been scrapped?

File: docs/perf_tips.md [hunk 2]
```diff
@@ -138,8 +134,7 @@ misses, or similar factors, but they are less likely to be a concern.)
 
 ## 7. Keep memory use and timeouts in check
 
-If you have increased the `-m` or `-t` limits more than truly necessary, consider
-dialing them back down.
+Consider setting low values for -m and -t.
```
Comment (domenukk) [inline docs/perf_tips.md]: In backticks?

**Review by domenukk — APPROVED** [review]

File: docs/Changelog.md (hunk 1 above)
Comment (vanhauser-thc) [reply docs/Changelog.md:33]: error on my side to add this

File: docs/perf_tips.md (hunk 2 above)
Comment (vanhauser-thc) [reply docs/perf_tips.md]: done

File: src/afl-fuzz-mutators.c [hunk 3]
```diff
@@ -339,7 +339,7 @@ u8 trim_case_custom(afl_state_t *afl, struct queue_entry *q, u8 *in_buf,
 
   }
 
-  while (afl->stage_cur < afl->stage_max) {
+  while (afl->stage_cur <= afl->stage_max) {
```
Comment (vanhauser-thc) [inline src/afl-fuzz-mutators.c]: can someone have a look here? from our documentation we tell to return fir trim_post 1 if more trims can be done, resetting stage_max to 1, but stage_cur is 1 after the first loop ... what do?

**Comment by WorksButNotTested** [comment]
> @WorksButNotTested by trying to resolve merge conflicts, this file appeared from stable. Is it supposed to be there, or not?
> Sorry :)
@domenukk This python script has been replaced with a shell script in the `dev` branch to reduce FRIDA mode's build dependencies.
Several instances were previously in the repo, they have now been consolidated down to the single bash script instance.

**Review by hexcoder- — APPROVED** [review]

**Comment by hexcoder-** [comment]
Good planning to push now to stable! Makes our lives probably easier later.

**Review by WorksButNotTested — CHANGES_REQUESTED** [review]
Awesome. Thanks for sorting that.

**Pull request merged**